input/papers/1999/1999AA...350..757A/tev-000049-lc.ecsv

```diff
@@ -0,0 +1,33 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: time, unit: MJD, datatype: float64}
+# - {name: flux, unit: Crab, datatype: float32}
+# - {name: flux_err, unit: Crab, datatype: float32}
+# meta: !!omap
+# - COMMENT: [Mrk 421]
```

Comment:
My suggestion would be that we put comments like here:
https://github.com/gammapy/gamma-cat/blob/b103005c0e1f3b68c9d55f6664da6260f7ea3ecc/input/papers/2008/2008A%2526A...477..353A/tev-000093.ecsv#L12

I.e. call the field `comments`, and use one (possibly multi-line) string, not a list.

OK?

Also, if it's just about the source name, my suggestion would be that we either don't put it input files at all.
Or if you want to put it because you think it's useful for maintenance, then let's put it like this:
https://github.com/gammapy/gamma-cat/blob/b103005c0e1f3b68c9d55f6664da6260f7ea3ecc/input/papers/2008/2008A%2526A...477..353A/tev-000093.ecsv#L10

You can go to https://gammapy.github.io/gamma-cat/sources.html and find the `common_name` for your sources. In this case it's `Markarian 421`, which comes from here: https://github.com/gammapy/gamma-cat/blob/master/input/sources/tev-000049.yaml#L3